extras/homeassistant/custom_components/open_filament_sensor/__init__.py

```
@@ -0,0 +1,86 @@
+        """Fetch data from OFS device."""
+        try:
+            async with asyncio.timeout(10):
+                async with self.session.get(self._url) as response:
+                    if response.status != 200:
+                        raise UpdateFailed(f"HTTP error {response.status}")
+                    data = await response.json()
+                    return data
+        except aiohttp.ClientError as err:
+            raise UpdateFailed(f"Error communicating with OFS: {err}") from err
+        except asyncio.TimeoutError as err:
+            raise UpdateFailed(f"Timeout communicating with OFS") from err
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)